src/System.Management.Automation/engine/Utils.cs

```diff
@@ -895,113 +895,49 @@ internal static bool IsAdministrator()
 #endif
         }
 
-        internal static bool NativeItemExists(string path)
+        internal static bool ItemExists(string path)
         {
-            bool unusedIsDirectory;
-            Exception unusedException;
-
-            return NativeItemExists(path, out unusedIsDirectory, out unusedException);
+            bool result = false;
+            try
+            {
+                result =  ItemExists(path, out bool unused);
```

Comment:
extra whitespace after equals sign

Reply:
perhaps use named parameter to make it more clear what `unused` is?

Reply:
`unused` here is self explaination - we don't use the parameter at all.

Reply:
Extra space fixed.